From: https://github.com/drforr/perl6-readline/issues/23

On Slackware 14.2, symbols from ncurses need to be loaded before symbols from
readline can be used.

--- lib/Readline.pm.orig	2017-09-21 10:14:05.000000000 +0200
+++ lib/Readline.pm	2018-07-02 22:14:26.473664202 +0200
@@ -1,5 +1,11 @@
 use v6;
 use NativeCall;
+
+INIT {
+    my sub tgetnum(Str) returns int32 is native('ncurses') { * }
+    tgetnum('');
+}
+
 #
 # XXX &cglobal.signature -> (Any $libname, Any $symbol, Any $target-type)
 #
